packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.js

```diff
@@ -102,17 +122,47 @@ export default class QuanticNumericFacet extends LightningElement {
 
   /** @type {NumericFacetState} */
   @track state;
+  /** @type {NumericFilterState} */
+  @track stateFilter = {
+    isLoading: false,
+    facetId: undefined
+  }
 
   /** @type {NumericFacet} */
   facet;
-  /** @type {Function} */
+  /**  @type {import("coveo").NumericFilter} */
```

Comment:
The type imports need to be declared as `@typedef` before the class declaration. See other types in file for examples.